internal/controllers/image_controller.go

```diff
@@ -235,7 +242,7 @@ type fetchAuthResponse struct {
 	Key string `json:"key"`
 }
 
-func (r *ImageReconciler) fetchAuth(ctx context.Context, log logr.Logger) (string, string, error) {
```

Comment:
Can we please keep the `ctx`? Currently, a lot of the `ceph` commands do not support a `ctx`, however libraries tend to add a `ctx` from time to time. If we run into issues we can also implement our cancelation logic.

Reply:
reverted back changes